sources/platform/management/subscription/billing.md

```diff
@@ -87,6 +87,17 @@ Please note that any changes due to your plan downgrade will be effective immedi
 
 Your new price will be reflected in your next invoice.
 
+### Enabling cache
+You can upgrade your dyamic SKU to have `cache` addon. This will make your builds faster. To know more about caching click here.
+
+* You can purchase cache addon for any of your existing dynamic SKU or new dynamic SKU by clicking on the cache checkbox in your SKU row.
+
+* You will be charged some fixed amount for enabling caching apart from your SKU charge.
```

Comment:
"Node caching is a chargeable add-on. When you enable caching for a node, the unit price will be updated accordingly.